airflow/providers/apache/hive/CHANGELOG.rst

```diff
@@ -27,6 +27,14 @@
 Changelog
 ---------
 
+6.1.4
+.....
+
+Misc
+~~~~
+
+* ``Bring back mysql-connector-python as required depednency (#32989)``
```

Comment:
```suggestion
* ``Bring back mysql-connector-python as required dependency (#32989)``
```

Reply:
We can not change commit messages in provider releases :(

Reply:
We can change them in CHANGELOG in fact. We can't change them in commits.rst but in changelog they are matched by PR # not the message.